internal/resources/resource_volume.go

```diff
@@ -136,6 +138,20 @@ func volumeSchema() map[string]*schema.Schema {
 			Computed:    true,
 			Description: "The storage pool of the volume to be created.",
 		},
+
+		vCollection: {
+			Type:        schema.TypeString,
+			Required:    false,
+			Optional:    true,
+			Description: "The volume collection of the volume to be created.",
+		},
+
+		vCollectionID: {
```

Comment:
should this be similar to vStoragePoolID? see Computed: true

Reply:
I am allowing to specify either name or ID. Is that a problem?

Reply:
just saying it differs from other optional ID fields so thought we should be consistent.
Marking as Computed, it can still be provided . Per docs: 
"Computed indicates whether the provider may return its own value for this attribute or not. Computed cannot be used with Required. If Required and Optional are both false, the attribute will be considered "read only" for the practitioner, with only the provider able to set its value."

Reply:
I am saying, it is not computed because I allow the user to enter the value in.
It provides the function to create volume- either the user enters name or id.
I do not see the need to change this. 
I dont see there is any consistency issue

Reply:
marking it Computed still allows user to enter it if desired, expectation is probably they will use the name (not the id) in which case the id will be computed

Reply:
it does not make any differenernce sonce the filed is optional anyway. 
If you are bent upon debating this and debating I will change it - I do not see any technical need to change it for the  record .